src/js/components/scene-selector.jsx

```diff
@@ -66,22 +69,24 @@ var SceneSelector = React.createClass({
 
 //scene actions
   _createSceneHandler: function(sceneName) {
-    HubSendActions.tryCreateScene(sceneName) //dosn't check for duplicates!
+    var self = this;
+    HubSendActions.tryCreateScene(sceneName, function(newScene) {
+      self.setState({sceneListFilter: {value: newScene, label: newScene.name}})
+    }) //dosn't check for duplicate names!
```

Comment:
Personally I think scene names are fine not being unique, especially with our final goal of a SaaS approach with solution and projects.  We will always have a scene id and other indexing opportunities.